x/ibc/07-tendermint/client/cli/tx.go

```diff
@@ -33,11 +33,13 @@ const flagTrustLevel = "trust-level"
 // in https://github.com/cosmos/ics/tree/master/spec/ics-002-client-semantics#create
 func GetCmdCreateClient(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "create [client-id] [path/to/consensus_state.json] [trusting_period] [unbonding_period] [max_clock_drift]",
-		Short:   "create new tendermint client",
-		Long:    "create new tendermint client. Trust level can be a fraction (eg: '1/3') or 'default'",
-		Example: fmt.Sprintf("%s tx ibc client create [client-id] [path/to/consensus_state.json] [trusting_period] [unbonding_period] [max_clock_drift] --trust-level default --from node0 --home ../node0/<app>cli --chain-id $CID", version.ClientName),
-		Args:    cobra.ExactArgs(6),
+		Use:   "create [client-id] [path/to/consensus_state.json] [trusting_period] [unbonding_period] [max_clock_drift]",
+		Short: "create new tendermint client",
+		Long:  "create new tendermint client. Trust level can be a fraction (eg: '1/3') or 'default'",
+		Example: fmt.Sprintf(
```

Comment:
Have you tested how this will render?

Reply:
```
simcli tx ibc tendermint create -h
create new tendermint client. Trust level can be a fraction (eg: '1/3') or 'default'

Usage:
  simcli tx ibc tendermint create [client-id] [path/to/consensus_state.json] [trusting_period] [unbonding_period] [max_clock_drift] [flags]

Examples:
<appcli> tx ibc tendermint create [client-id] [path/to/consensus_state.json] [trusting_period] [unbonding_period] [max_clock_drift] --trust-level default --from node0 --home ../node0/<app>cli --chain-id $CID

Flags:
  -a, --account-number uint      The account number of the signing account (offline mode only)
  -b, --broadcast-mode string    Transaction broadcasting mode (sync|async|block) (default "sync")
      --dry-run                  ignore the --gas flag and perform a simulation of a transaction, but don't broadcast it
      --fees string              Fees to pay along with transaction; eg: 10uatom
      --from string              Name or address of private key with which to sign
      --gas string               gas limit to set per-transaction; set to "auto" to calculate required gas automatically (default 200000) (default "200000")
      --gas-adjustment float     adjustment factor to be multiplied against the estimate returned by the tx simulation; if the gas limit is set manually this flag is ignored  (default 1)
      --gas-prices string        Gas prices to determine the transaction fee (e.g. 10uatom)
      --generate-only            Build an unsigned transaction and write it to STDOUT (when enabled, the local Keybase is not accessible)
  -h, --help                     help for create
      --indent                   Add indent to JSON response
      --keyring-backend string   Select keyring's backend (os|file|kwallet|pass|test) (default "os")
      --ledger                   Use a connected Ledger device
      --memo string              Memo to send along with transaction
      --node string              <host>:<port> to tendermint rpc interface for this chain (default "tcp://localhost:26657")
      --offline                  Offline mode (does not allow any online functionality
  -s, --sequence uint            The sequence number of the signing account (offline mode only)
      --trust-level string       light client trust level fraction for header updates (default "default")
      --trust-node               Trust connected full node (don't verify proofs for responses) (default true)
  -y, --yes                      Skip tx broadcasting prompt confirmation
```

Reply:
IMHO I think it had better to have `[path/to/consensus_state.json]` as last argument

Reply:
> IMHO I think it had better to have [path/to/consensus_state.json] as last argument

👍 I'll update all the commands in another PR